### PR TITLE
Sorted strings in strings.xml by name (Fixed #1124)

### LINKED ISSUE #1124: Sort strings resource values in strings.xml by name
**Actual Behaviour**

Currently, the string resource values have been randomly placed in the strings.xml file which makes it a bit difficult to look for a string value. We need to scroll through the entire list or use search to look for desired value.

**Expected Behaviour**

It would be better if the strings were arranged alphabetically so that it would be easier to look up a string using its label.

**Screenshots of the issue**

strings_file

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by arundhati24**
Sorted strings resource values present in strings.xml by name using
AndroidXmlSorter plugin.

Fixes #1124 

**Screenshots for the change:** 
strings_arranged

### CODE REVIEW

**Comment by arundhati24** [comment]
@chiragw15  Please review this PR. :slightly_smiling_face:

**Comment by arundhati24** [comment]
@chiragw15 I made the required changes. Also, I made similar changes in #1132 . Please review.

**Comment by arundhati24** [comment]
@chiragw15 I have resolved conflicts and have fixed the failing CIrcle CI builds. It would be great if you could please review the changes? :slightly_smiling_face:

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,209 +1,304 @@
...
+
+    <string name="action_select_language">Select a Language</string>
+    <string name="action_select_query_language">Select a query language</string>
+    <string name="action_select_tts_language">Select a text to speech engine language</string>
+
     <string name="activity_login_forgot_pass">Forgot Password?</string>
+    <string name="activity_login_imageview_description" translatable="false">SUSI Logo</string>
+    <string name="activity_login_log_in">Log in</string>
     <string name="activity_login_sign_up">Sign up for SUSI</string>
+    <string name="activity_login_sing_up">Sign Up</string>
```
Comment (akshatnitd) [inline app/src/main/res/values/strings.xml]: `sing` ?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,209 +1,304 @@
...
+
+    <string name="app_name">SUSI AI</string>
+
     <string name="background_no_wall">no_wall</string>
+
+    <string name="content">content</string>
+
+    <string name="dialog_retrieve_messages_title">Retrieving old messages…</string>
+
+    <string name="email_invalid">The e-mail address you entered doesn\'t belong to an account.
```
Comment (akshatnitd) [inline app/src/main/res/values/strings.xml]: Remove extra spaces.

**Comment by iamareebjamal** [comment]
@arundhati24 Let's finalize the PR?

**Comment by iamareebjamal** [comment]
@arundhati24 Some comments addressed by reviewers are not resolved

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (arundhati24) [reply app/src/main/res/values/strings.xml]: @akshatnitd Fixed that..

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (akshatnitd) [reply app/src/main/res/values/strings.xml]: @arundhati24 I am talking about the space between `to an account.` and `Please check...`

Comment (arundhati24) [reply app/src/main/res/values/strings.xml]: @akshatnitd Fixed that.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,216 +1,304 @@
...
-    <string name="server_settings_title">Server Settings</string>
-    <string name="server_pref">Select Server</string>
-    <string name="server_select_summary">Select backend server for the app.</string>
-    <string name="search">Search</string>
-    <string name="select_background">Select background</string>
+    <string name="action_settings">Settings</string>
+    <string name="action_share">Share</string>
+    <string name="action_wall_settings">Wallpaper</string>
+
+    <string name="app_name">SUSI AI</string>
```
[the remark this answers is not in the document]
Comment (arundhati24) [reply app/src/main/res/values/strings.xml]: @iamareebjamal Done.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,216 +1,304 @@
...
-    <string name="error_occurred_try_again">An error occurred. Please try again.</string>
-    <string name="error_invalid_token">Invalid Access Token.</string>
-    <string name="error_internet_connectivity">Internet Connectivity Problem.</string>
     <string name="error_hotword">Sorry, hotword detection is not supported on your device.</string>
+    <string name="error_internet_connectivity">Internet Connectivity Problem.</string>
+    <string name="error_invalid_token">Invalid Access Token.</string>
+    <string name="error_occurred_try_again">An error occurred. please try again.</string>
     <string name="error_rating">Unable to rate skill. An error occurred.</string>
-    <string name="no_skill_name">Name not available</string>
+
```
Comment (batbrain7) [inline app/src/main/res/values/strings.xml]: @arundhati24 this is the reason for the crash of the app on scrolling down the settingsfragment.
@iamareebjamal please review

Comment (batbrain7) [reply app/src/main/res/values/strings.xml]: #1262

Comment (iamareebjamal) [reply app/src/main/res/values/strings.xml]: Please add a new PR fixing the issue

**Pull request merged**